src/main/java/de/javakaffee/kryoserializers/guava/ImmutableMapSerializer.java

```diff
@@ -54,5 +54,17 @@ public static void registerSerializers(final Kryo kryo) {
         kryo.register(ImmutableMap.of(o1, o1).getClass(), serializer);
         kryo.register(ImmutableMap.of(o1, o1, o2, o2).getClass(), serializer);
 
+        Map<DummyEnum,Object> enumMap = new EnumMap<DummyEnum, Object>(DummyEnum.class);
+        for (DummyEnum e : DummyEnum.values()) {
+            enumMap.put(e, o1);
```

Comment:
Ah, I just saw that `ImmutableEnumMap.asImmutable(EnumMap<K, V> map)` checks the size of `map`, at least 2 elements are needed so that an `ImmutableEnumMap` instance is created.